### PR TITLE
Tox testing

### PR DESCRIPTION
**Pull request by robin900**
Added tox.ini for 2.6-3.4. Added "smoke test suite" via unittest. It runs successfully for py26-py34 if you have a valid key/secret in tests/test_smoke.py. You will need to create a Mixpanel account with an email you control, seed some events in a project for that account, get the key/secret, and establish those in tests/test_smoke.py for automated tox testing in github.

### CODE REVIEW

**Comment by robin900** [comment]
Will re-try this PR after rebasing.

File: tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+
```
Comment (cooncesean) [inline tests/__init__.py]: Just noting the extra blank line.

File: tests/test_smoke.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import unittest
+import mixpanel_query.client
+import os
+
+class SmokeTestCase(unittest.TestCase):
```
Comment (cooncesean) [inline tests/test_smoke.py]: Docstring would be awesome :)

File: tests/test_smoke.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import unittest
+import mixpanel_query.client
```
Comment (cooncesean) [inline tests/test_smoke.py]: Just a note to PEP-8 these imports like so:

```python
import os
import unittest

import mixpanel_query.client


class SomeTestCase...
```

**Pull request closed without merging**